libpod/container_internal.go

```diff
@@ -522,7 +522,7 @@ func (c *Container) init(ctx context.Context) error {
 	}
 
 	// With the spec complete, do an OCI create
-	if err := c.runtime.ociRuntime.createContainer(c, c.config.CgroupParent); err != nil {
+	if err := c.runtime.ociRuntime.createOCIContainer(c, c.config.CgroupParent, false); err != nil {
```

Comment:
You don't need the other functions in createContainer?

Reply:
Not sure, this changed between my first PR and now (after rebase). I will have a closer look what is the right function.

Reply:
Using createContainer() also just works. It seems, however, nobody else is using the createContainer() function. Is that a function for external usage?

Reply:
I believe this is a rootless thing - if you have root, createContainer just wraps createOCIContainer, but for rootless it also does some additional work to ensure things are set up properly